tests/test_examples.py

```diff
@@ -8,6 +8,9 @@
 
 from tests.common import EXAMPLES_DIR, requires_cuda
 
+pytestmark = pytest.mark.stage('test_examples')
```

Comment:
Adding a common marker so that `test_coverage` is also included; otherwise it would run for other test stages too.